NitroxClient/GameLogic/InitialSync/RemotePlayerInitialSyncProcessor.cs

```diff
@@ -36,7 +36,7 @@ public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualW
 
                 List<TechType> equippedTechTypes = playerData.EquippedTechTypes.Select(techType => techType.ToUnity()).ToList();
                 RemotePlayer player = remotePlayerManager.Create(playerData.PlayerContext, equippedTechTypes);
-
+                bool swimming = true;
```

Comment:
Looks like you only call `IsSwimming` when they are in a subroot.  What about when they are not in a subroot but above elevation? (such as on top of the escape pod).  Should we always call the below function?